### PR TITLE
Introduce apiNamePrefix and apiNameSuffix config options #136

### PR DESCRIPTION
**Pull request by kobylynskyi**
@joffrey-bion could you please review this? We discussed it like a month ago and I finally got some time to do it.

I am targetting to release it in version 2.0.0 along with some other major features because we are introducing here a breaking change.

### CODE REVIEW

File: plugins/maven/graphql-java-codegen-maven-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/GraphQLCodegenMojo.java [hunk 1]
```diff
@@ -138,6 +144,8 @@ public void execute() throws MojoExecutionException {
         MappingConfig mappingConfig = new MappingConfig();
         mappingConfig.setPackageName(packageName);
         mappingConfig.setCustomTypesMapping(customTypesMapping != null ? customTypesMapping : new HashMap<>());
+        mappingConfig.setApiNamePrefix(apiNameSuffix);
```
Comment (joffrey-bion) [inline plugins/maven/graphql-java-codegen-maven-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/GraphQLCodegenMojo.java:147]: I believe there is a mistake here

File: plugins/gradle/graphql-java-codegen-gradle-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/gradle/GraphQLCodegenGradleTask.java [hunk 2]
```diff
@@ -73,6 +75,8 @@ public void generate() throws Exception {
         MappingConfig mappingConfig = new MappingConfig();
         mappingConfig.setPackageName(packageName);
         mappingConfig.setCustomTypesMapping(customTypesMapping);
+        mappingConfig.setApiNamePrefix(apiNameSuffix);
```
Comment (joffrey-bion) [inline plugins/gradle/graphql-java-codegen-gradle-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/gradle/GraphQLCodegenGradleTask.java:78]: Looks like there is the same prefix/suffix mistake here

File: plugins/maven/graphql-java-codegen-maven-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/GraphQLCodegenMojo.java (hunk 1 above)
Comment (kobylynskyi) [reply plugins/maven/graphql-java-codegen-maven-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/GraphQLCodegenMojo.java:147]: Oh man. Thanks for pointing this out!

Comment (kobylynskyi) [reply plugins/maven/graphql-java-codegen-maven-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/GraphQLCodegenMojo.java]: Done

File: plugins/gradle/graphql-java-codegen-gradle-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/gradle/GraphQLCodegenGradleTask.java (hunk 2 above)
Comment (kobylynskyi) [reply plugins/gradle/graphql-java-codegen-gradle-plugin/src/main/java/io/github/kobylynskyi/graphql/codegen/gradle/GraphQLCodegenGradleTask.java]: Thanks! Fixed

**Comment by kobylynskyi** [comment]
Thanks, @joffrey-bion!

**Pull request merged**